src/Lykke.Job.TxDetector/Sagas/ConfirmationsSaga.cs

```diff
@@ -124,5 +137,86 @@ public class ConfirmationsSaga
                 sender.SendCommand(sendNotificationCommand, "notifications");
             }
         }
+
+        private async Task Handle(ConfirmationSavedEvent evt, ICommandSender sender)
+        {
+            _log.WriteInfo(nameof(ConfirmationSavedEvent), evt, "");
```

Comment:
maybe add some Msg ... "Handle received message". It's can help for found all log by handle